lib/LaTeXML/Package/longtable.sty.ltxml

```diff
@@ -125,12 +125,22 @@ DefPrimitive('\@longtable@@toccaption{}', sub {
 
 #======================================================================
 # Not used, but must be defined.
-DefRegister('\LTLeft'      => Glue(0));
-DefRegister('\LTRight'     => Glue(0));
-DefRegister('\LTPre'       => Glue(0));
-DefRegister('\LTPost'      => Glue(0));
-DefRegister('\LTcapwidth'  => Glue(0));
-DefRegister('\LTchunksize' => Number(20));
+DefRegister('\LTleft'      => Glue(0, '1fill'));
+DefRegister('\LTright'     => Glue(0, '1fill'));
```

Comment:
Oooh, bad cargo culting; this is going to fail after #1724. Should write `Glue('0pt plus 1fill')`.

Reply:
Exactly as you say, and good that #1724 cleans that up too!